accountsplus/forms.py

```diff
@@ -0,0 +1,7 @@
+import django.forms
```

Comment:
@nasief its a good practice to add
`from __future__ import unicode_literals`
At the top of every python files